code/game/machinery/computer/communications.dm

```diff
@@ -130,7 +130,7 @@
 				cooldown_message = world.time
 
 		if("award")
-			print_medal(usr, src)
+			open_medal_panel(usr, src)
```

Comment:
Note: As far as I can tell this is impossible to access atm because the only communications computer on the almayer is in  the secure command hallway and that doesn't satisfy the `if(!istype(loc.loc, /area/almayer/command/cic)) //Has to be in the CIC. Can also be a generic CIC area to communicate, if wanted.` test.

Reply:
this is just a Ctrl-F replace. I can change it if you want, but seems sorta out-of-scope

Reply:
Its just a note; I'm not sure why it is how it is and why literally no one has ever reported this problem before.